x/staking/types/delegation.go

```diff
@@ -315,14 +316,21 @@ func (d Redelegation) String() string {
   Delegator:                 %s
   Source Validator:          %s
   Destination Validator:     %s
-  Entries:`, d.DelegatorAddress, d.ValidatorSrcAddress, d.ValidatorDstAddress)
+  Entries:
+`,
+		d.DelegatorAddress, d.ValidatorSrcAddress, d.ValidatorDstAddress,
+	)
+
 	for i, entry := range d.Entries {
-		out += fmt.Sprintf(`    Redelegation %d:
+		out += fmt.Sprintf(`    Redelegation Entry #%d:
       Creation height:           %v
       Min time to unbond (unix): %v
-      Dest Shares:               %s`, i, entry.CreationHeight,
-			entry.CompletionTime, entry.SharesDst)
+      Dest Shares:               %s
+`,
```

Comment:
ditto

Reply:
See above.

Reply:
...this would print it. Easy solution: strip the last `\n` character from the string before returning it

Reply:
Ok, yeah I can strip the last newline (from the last redelegation entry), sure.

Reply:
Updated.